PrismLauncher/PrismLauncher
# Add Modpack-page to instances that were created from files

Parrent PR: https://github.com/PrismLauncher/PrismLauncher/pull/1405
should fix: https://github.com/PrismLauncher/PrismLauncher/issues/1394#issuecomment-1767449001
From that comment looks like I did not fix the original issue just extended the functionality of the managed packs(downloaded through prism from modrinth/curseforge)
This PR aims to have a page for local packs (imported from files).
This only works with CurseForge and Modrinth packs.

How to test:

    download a modrinth pack( an older version)
    install it using the mrpack file
    download a newer version of the pack(outside of prism)
    click edit instance on the created instance
    you should have a similar page as when that pack would be downloaded through prism
    entering that page you will be greeted with:
        the pack name(the name of the instance)
        a big button with upload from file
        a changelog that explains this is a local pack and can only be updated through respective format
    click on the upload from file
    select the new version
    profit

Also fixes #1731
fixes #1813
